contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -3477,6 +3477,15 @@ pltsql_store_func_default_positions(ObjectAddress address, List *parameters, con
 
 	if (HeapTupleIsValid(oldtup))
 	{
+		char* alter_str = strtok(original_query, " ");
+		if(alter_str && strncasecmp("alter", alter_str, 5) == 0) {
+			char* proc_str = strtok(NULL, " ");
+			if(proc_str && strncasecmp("proc", proc_str, 4) == 0) {
+				/* Modify the original string definiton for information_schema.routines */
+				char* newQuery = psprintf("CREATE %s", (queryString + 6));
```

Comment:
I don't think this is the right approach. It will wipe out some comments in the original `create` query, e.g.,
```
create /* some comments */   procedure alter_proc_p1 as select 1;
```
, and it may not be able to handle comments in the `alter` query, e.g.,
```
alter -- another comment
    procedure alter_proc_p1 as select 2;
```
```